apis/v1alpha1/iam_response.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import "cloud.google.com/go/iam/apiv1/iampb"
+
+// IAMResponse contains the IAM policy returned and its resource information.
+type IAMResponse struct {
+	// IAM policy of the resource.
+	Policy *iampb.Policy
+
+	// Resource represents one of GCP organization, folder, and project.
+	Resource string
+}
```

Comment:
Forgot to ask - How do we plan to make use of this?

Reply:
Direct use of it is in unit test, besides it make sense to return the new policy after updating similar to the go client SetIamPolicy which also returns the new policy.

Reply:
discussed offline